pkg/config/config.go

```diff
@@ -30,6 +30,7 @@ type Config struct {
 	KeyFile        string             `yaml:"key_file"`
 	Keys           map[string]string  `yaml:"keys"`
 	LogLevel       string             `yaml:"log_level"`
+	AuthType	   string			  `yaml:"auth_type"`
```

Comment:
```suggestion
	KeyProvider	   string			  `yaml:"key_provider"`
```

can you also run go fmt / goimports on files in this commit?